projects/avahi/avahi_packet_consume_key_fuzzer.cc

```diff
@@ -1,34 +0,0 @@
-#include <stdint.h>
```

Comment:
I didn't copy the fuzz targets verbatim but just in case @alex @jonathanmetzman I wonder if it would be OK if I relicensed them using https://github.com/lathiat/avahi/blob/6395c45bcd1210cbbb23fc92294fd61e968daaf2/docs/HACKING#L60. Without that they can't be kept upstream.

Reply:
My contributions to the avahi fuzzer (which was the original authorship, I believe) may be used under _either_ the terms the Apache v2 License, the BSD 2-clause license, or the Avahi project's licence.

That shoudl let you do what you need.

Reply:
Thanks @alex !